### DISCUSSION TITLE
Is it possible to get a BBC Micro emulator running?

### DISCUSSION THREAD

**Discussion by SuperBadger77** [Rparadise-Team/Koriki#16] [Ideas & Requests]
Is is possible to use the same emulator from the Adam image?

**Comment by eduardofilo** [comment]
Someone will have to port it. As far as I can see, it could start from the Linux version:
http://www.mkw.me.uk/beebem/

Beeing an SDL project, it seems feasible, as was the case with RG350. Perhaps that port is a better starting point.
https://github.com/jeremyrayner/beebem-rg350/

**Reply by SuperBadger77** [reply]
Are there any guides on porting it to this device? I looked at doing this for ArkOS, could try and do it for this machine if you let me know the target architecture?

**Reply by eduardofilo** [reply]
Here are the specifications:
https://steward-fu.github.io/website/handheld/miyoo-mini/review.htm

The key codes:
https://steward-fu.github.io/website/handheld/miyoo-mini/sdlkey.htm

I use this toolchain to build projects:
https://github.com/shauninman/union-miyoomini-toolchain
